### ISSUE TITLE
Make it clearer that tests are clickable, and that test cases have no "global" instance until you click on a test

### ISSUE THREAD

**Issue by gregsdennis** [bowtie-json-schema/bowtie#318]
The instance isn't printing in the test details

<img width="972" alt="image" src="https://github.com/bowtie-json-schema/bowtie/assets/2676804/4f508fa0-0212-4a4a-b644-e9fdd046cc9b">

**Comment by Julian** [comment]
Does it not populate when you click on a test?

**Comment by gregsdennis** [comment]
I'm not sure what you mean. In the screenshot, I've expanded one of the tests.

**Comment by Julian** [comment]
I agree it's unintuitive, not sure we have a replacement design yet though -- eventually (soon?) I think we'll want to remove the entirety of the details section below in favor of something dynamically rendered (perhaps in a sidebar).

But yeah that thing you expanded corresponds to a test case as opposed to an individual test, as in in the suite we have a group of tests and they all use the same schema, but of course each has its own instance.

Suggestions certainly welcome.

**Comment by Julian** [comment]
It's also super confusing here that clicking on a test does not scroll the view, so if the "Instance" box is scrolled offscreen above, there is 0 indication something has changed when clicking on a test.

**Comment by gregsdennis** [comment]
An option for this could be hint text in the instance cell that says something like, "Click a test case below to view the instance."
